docs/docs/page-through-results.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Page through Results
```

Comment:
I tried to make most topics use active verbs for parallel structure--so Get Started, Filter and Sort, Page through Results.

My preference is that we use page as the verb, pagination when referring to options. (No super strong feelings though.)